doc/source/ray-observability/getting-started.rst

```diff
@@ -95,7 +97,7 @@ Tasks and actors are grouped and nested by the following criteria.
 
 .. note::
 
-  Ray dashboard can only display or retrieve up to 10K tasks at a time. If there are more than 10K tasks from your job,
+  Job detail page can only display or retrieve up to 10K tasks per job. If there are more than 10K tasks from your job,
```

Comment:
```suggestion
  The Job Detail page can only display or retrieve up to 10K tasks per job. For jobs with more than 10K tasks,
```

Reply:
What I want to convey here is: "For the portion of tasks that exceed the 10K limit per job, they are unaccounted."

Reply:
I see, then you described it well: 
"For Jobs with more than 10K Tasks, the portion of Tasks that exceed the 10K limit are unaccounted."